src/chess_ai/example.py

```diff
@@ -0,0 +1,185 @@
+    -6, 13, 13, 26, 34, 12, 10, -9,
+    0, 15, 15, 15, 14, 27, 18, -18,
+    4, 15, 16, 0, 7, 21, 33, -1,
+    -19, -7, -37, -26, -36, -14, 2, -22,
+]
+
+eg_bishop_table = [
+    -14, -21, -11, -8, -7, -9, -17, -24,
+    -8, -4, 7, -12, -3, -13, -4, -14,
+    2, -8, 0, 11, 2, 0, -5, -3,
+    -3, 9, 12, 9, 14, 10, 3, 2,
+    -6, 3, 13, 19, 7, 10, -3, -9,
+    -12, 0, 0, 12, 10, 2, 0, -19,
+    -14, -15, -2, -5, -1, -10, -20, -22,
+    -22, -9, -23, -5, -9, -16, -5, -17,
+]
+
+mg_rook_table = [
+    32, 42, 32, 51, 63, 9, 31, 43,
+    27, 32, 58, 62, 80, 67, 26, 44,
+    -5, 19, 26, 36, 17, 45, 61, 16,
+    -24, -11, 7, 26, 24, 35, -8, -20,
+    -36, -26, -12, -1, 9, -7, 6, -23,
+    -45, -25, -16, -17, 3, 0, -5, -33,
+    -44, -16, -20, -9, -1, 11, -6, -71,
+    -19, -13, 1, 17, 16, 7, -37, -26,
+]
+
+eg_rook_table = [
+    13, 10, 18, 15, 12, 12, 8, 5,
+    11, 13, 13, 11, -3, 3, 8, 3,
+    7, 7, 7, 5, 4, -3, -5, -3,
+    4, 3, 13, 1, 2, 1, -1, 2,
+    3, 5, 8, 4, -5, -6, -8, -11,
+    -4, 0, -5, -1, -7, -12, -8, -16,
+    -6, -6, 0, 2, -9, -9, -11, -3,
+    -9, 2, 3, -1, -5, -13, 4, -20
+]
+
+mg_queen_table = [
+    -28, 0, 29, 12, 59, 44, 43, 45,
+    -24, -39, -5, 1, -16, 57, 28, 54,
+    -13, -17, 7, 8, 29, 56, 47, 57,
+    -27, -27, -16, -16, -1, 17, -2, 1,
+    -9, -26, -9, -10, -2, -4, 3, -3,
+    -14, 2, -11, -2, -5, 2, 14, 5,
+    -35, -8, 11, 2, 8, 15, -3, 1,
+    -1, -18, -9, 10, -15, -25, -31, -50
+]
+
+eg_queen_table = [
+    -9, 22, 22, 27, 27, 19, 10, 20,
+    -17, 20, 32, 41, 58, 25, 30, 0,
+    -20, 6, 9, 49, 47, 35, 19, 9,
+    3, 22, 24, 45, 57, 40, 57, 36,
+    -18, 28, 19, 47, 31, 34, 39, 23,
+    -16, -27, 15, 6, 9, 17, 10, 5,
+    -22, -23, -30, -16, -16, -23, -36, -32,
+    -33, -28, -22, -43, -5, -32, -20, -41
+]
+
+mg_king_table = [    -65,  23,  16, -15, -56, -34,   2,  13, 
+                     29,  -1, -20,  -7,  -8,  -4, -38, -29,  
+                     -9,  24,   2, -16, -20,   6,  22, -22,  
+                     -17, -20, -12, -27, -30, -25, -14, -36,   
+                     -49,  -1, -27, -39, -46, -44, -33, -51, 
+                     -14, -14, -22, -46, -44, -30, -15, -27, 
+                     1,   7,  -8, -64, -43, -16,   9,   8, 
+                     -15,  36,  12, -54,   8, -28,  24,  14,]
+
+eg_king_table = [  -74, -35, -18, -18, -11,  15,   4, -17,
+                     -12,  17,  14,  17,  17,  38,  23,  11,
+                      10,  17,  23,  15,  20,  45,  44,  13,
+                    -8,  22,  24,  27,  26,  33,  26,   3, 
+                    -18,  -4,  21,  24,  27,  23,   9, -11,
+                     -19,  -3,  11,  21,  23,  16,   7,  -9,
+                     -27, -11,   4,  13,  14,   4,  -5, -17, 
+                       -53, -34, -21, -11, -28, -14, -24, -43,]
+
+
+mg_peice_values = [82, 337, 365, 477, 1025, 0] # [pawn, knight, bishop, rook , queen , king ] I think I am not sure
+eg_value = [94, 281, 297, 512, 936, 0]  #[pawn, knight, bishop, rook , queen , king ]
```

Comment:
Why is the value for the king `0`? Usually you would have him basically infinity (or negative infinity).